tools/hcluster_sg_parser/hcluster_sg_parser.xml

```diff
@@ -4,26 +4,25 @@
 <![CDATA[
 perl $__tool_directory__/hcluster_sg_parser.pl
 $inputFile
-> $report
 ]]>
     </command>
     <inputs>
         <param name="inputFile" type="data" format="tabular" label="hcluster output file in 3-column format" help="3-columns format: cluster_id cluster-size cluster-members" />
     </inputs>
     <outputs>
-        <data format="txt" name="report" hidden="true">
-            <discover_datasets pattern="(?P&lt;designation&gt;.+)_output\.txt" visible="true" />
-        </data>
+        <collection name="ids_lists" type="list" label="${tool.name} on ${on_string}">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)_output\.txt" format="txt" />
+        </collection>
     </outputs>
     <tests>
         <test>
             <param name="inputFile" ftype="tabular" value="hcluster_sg.tabular" />
-            <output name="report" file="empty_file" hidden="true">
+            <output_collection name="ids_lists" type="list">
                 <discovered_dataset designation="0" file="0_output.txt" />
```

Comment:
This probably needs to be changed - want to make sure this test is still actually testing something.

If not I think these need to be ``element``s not ``discovered_dataset``s. https://github.com/galaxyproject/galaxy/blob/dev/test/functional/tools/collection_creates_dynamic_nested.xml#L39